### PR TITLE
New property

### PR DESCRIPTION
**Pull request by Shimin-Zhang**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -110,7 +115,7 @@ app.post('/projects', function(req, resp) {
 			, website: req.body.website || ''
 			, picture: req.body.picture || ''
 			, statusCategory: determineStatusCategory(req.body.status) || ''
-			, coordinates: [coords.latitude, coords.longitude] || []
+			, coordinates: [(coords.latitude).toString(), (coords.longitude).toString()] || []
```
Comment (jmcelroy5) [inline server.js:155]: just curious why you had to .toString() these?

**Comment by jmcelroy5** [comment]
Woohoo, great work! =D Ready to merge once it's rebased

File: assets/new-hearing-form.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            </div>
+            <div class="col-xs-4">
+              <label for="start-time">Start Time</label>
+              <input type="time" name="start-time" value="">
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-xs-4">
+              <label for="hearing-type">Hearing Type </label>
+              <select class="" name="hearing-type">
```
Comment (jmcelroy5) [inline assets/new-hearing-form.html:37]: Oh, there are actually some fields we should add to this form - See @dbarnhart's updated schema in server.js

File: server.js (hunk 1 above)
Comment (Shimin-Zhang) [inline server.js:155]: not sure, everything in the database is in string format, changed it while getting the form working, might not be needed.

Comment (jmcelroy5) [inline server.js:155]: ok, as long as it doesn't break the map then we're probably fine :)

**Pull request merged**